test/t8_IO/t8_gtest_vtk_reader.cxx

```diff
@@ -45,37 +45,40 @@ class vtk_reader : public testing::TestWithParam<std::tuple<int, int, int>>{
     void SetUp() override{
       int mpiret = sc_MPI_Comm_size (sc_MPI_COMM_WORLD, &mpisize);
       SC_CHECK_MPI (mpiret);
+      mpiret = sc_MPI_Comm_rank (sc_MPI_COMM_WORLD, &mpirank);
+      SC_CHECK_MPI (mpiret);
       if (mpisize > T8_VTK_TEST_NUM_PROCS) {
         GTEST_SKIP ();
       } 
       file = std::get<0>(GetParam());
       file_type = gtest_vtk_filetypes[file];
       partition = std::get<1>(GetParam());
       main_proc = std::get<2>(GetParam());
-      if((file_type & VTK_PARALLEL_POLYDATA_FILE )&& partition){
-        GTEST_SKIP();
-      }
+      distributed = (file_type & VTK_PARALLEL_FILE) && partition;
```

Comment:
Isn't this a cool line of code? It reads like a sentence and states expliclitly what the author wants to express here. :)

Reply:
It is, indeed!